File: app/src/main/java/com/bq/zowi/models/ConnectionSuccessData.java
package com.bq.zowi.models;

/* JADX INFO: loaded from: classes.dex */
public class ConnectionSuccessData {
    private float batteryLevel;
    private String zowiAppId;
    private ZowiName zowiName;

    public ConnectionSuccessData(String zowiName, String zowiAppId, float batteryLevel) {
        this.zowiName = new ZowiName(zowiName);
        this.zowiAppId = zowiAppId;
        this.batteryLevel = batteryLevel;
    }

    public String getZowiName() {
        return this.zowiName.toString();
    }

    public String getZowiAppId() {
        return this.zowiAppId;
    }

    public float getBatteryLevel() {
        return this.batteryLevel;
    }
}

File: zowi-core/src/main/kotlin/com/bq/zowi/models/ConnectionSuccessData.kt
package com.bq.zowi.models

class ConnectionSuccessData(
    zowiName: String,
    val zowiAppId: String,
    val batteryLevel: Float
) {
    private val zowiName: ZowiName = ZowiName(zowiName)

    fun getZowiName(): String = zowiName.toString()
}
